src/calculateRentalCost.js

```diff
@@ -1,10 +1,19 @@
 /**
- * @param {number} days
- *
- * @return {number}
+ * Calculates the rental cost based on the number of days.
+ * @param {number} days - The number of days the car is rented.
+ * @return {number} - The total rental cost.
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50; // Apply $50 discount for 7 or more days
+  } else if (days >= 3) {
+    totalCost -= 20; // Apply $20 discount for 3 to 6 days
+  }
```

Comment:
According to the checklist, prefer using 'if' with 'return' over 'if else' to simplify later conditions. You can remove the 'else' and directly return the 'totalCost' after the first 'if' block.